--- a/src/runtime/platform/LibraryLoader.cs
+++ b/src/runtime/platform/LibraryLoader.cs
@@ -36,7 +36,7 @@ namespace Python.Runtime.Platform
         private static int RTLD_NOW = 0x2;
         private static int RTLD_GLOBAL = 0x100;
         private static IntPtr RTLD_DEFAULT = IntPtr.Zero;
-        private const string NativeDll = "libdl.so";
+        private const string NativeDll = "libdl.so.2";
 
         public IntPtr Load(string dllToLoad)
         {
